src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { slug = '' } = useParams();
+
+  const handleGettingPeople = async () => {
+    setIsLoading(true);
+
+    try {
+      const peopleFromServer = await getPeople();
+
+      setIsError(false);
+
+      const updatedPeople = peopleFromServer.map(person => (
+        {
+          ...person,
+          mother: peopleFromServer
+            .find(mother => mother.name === person.motherName),
+          father: peopleFromServer
+            .find(father => father.name === person.fatherName),
+        }
+      ));
+
+      setPeople(updatedPeople);
+    } catch (error) {
+      setIsError(true);
+    }
+
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
+    handleGettingPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+
+          {(isLoading && (<Loader />))}
```

Comment:
Probably redundant brackets (not 100% sure you need them)

Reply:
The ones around `isLoading && (<Loader />)`

Reply:
Thank you! 😊
Just forgot to remove them after playing with conditional rendering.